app/javascript/orangelight/availability.es6

```diff
@@ -35,25 +35,24 @@ export default class AvailabilityUpdater {
     // a search results page or a call number browse page
     if ($(".documents-list").length > 0) {
       const bib_ids = this.record_ids();
-      if (bib_ids.length < 1) { return; }
-      url = `${this.bibdata_base_url}/bibliographic/availability.json?bib_ids=${bib_ids.join()}`;
-      return $.getJSON(url, this.process_results_list)
-        .fail((jqXHR, textStatus, errorThrown) => {
-          if (jqXHR.status == 429) {
-            if (allowRetry) {
-              console.log(`Retrying availability for records ${bib_ids.join()}`);
-              window.setTimeout(() => {
-                this.update_availability_retrying();
-                this.request_availability(false);
-              }, 1500);
-            } else {
-              console.error(`Failed to retrieve availability data for bibs (retry). Records ${bib_ids.join()}: ${errorThrown}`);
-              this.update_availability_undetermined();
-            }
-            return;
-          }
-          return console.error(`Failed to retrieve availability data for bibs. Records ${bib_ids.join(", ")}: ${errorThrown}`);
-        });
+      if (bib_ids.length < 1) {
+        return;
+      }
+
+      const batch_size = 10;
+      const batches = this.ids_to_batches(bib_ids, batch_size);
+      console.log(`Requested at ${new Date().toISOString()}, batch size: ${batch_size}, batches: ${batches.length}, ids: ${bib_ids.length}`);
+
+      for(var i= 0; i < batches.length; i++) {
+        var batch_url = `${this.bibdata_base_url}/bibliographic/availability.json?bib_ids=${batches[i].join()}`;
+        console.log(`batch: ${i}, url: ${batch_url}`);
+        $.getJSON(batch_url, this.process_results_list)
+          .fail((jqXHR, _textStatus, errorThrown) => {
+            // Log that there were problems fetching a batch. Unfortunately we don't know exactly
+            // which batch so we cannot include that information.
+            console.error(`Failed to retrieve availability data for batch. HTTP status: ${jqXHR.status}: ${errorThrown}`);
+          });
```

Comment:
Notice that I removed the retry logic for HTTP 429 (too many requests) but this kind of error does not seem to have been triggered (at least it does not show in our logs) and removing it should not be an issue. Additionally, even if we get an error, with this new version the error will be limited to a given batch not to the entire set of MMS_IDs on the page. 

Plus, the batch logic now prevents the of requesting more than 100 MMS_IDs at once from Alma -- and this error was indeed in our logs.